src/main/pages/che-7/hands-on-guidance/proc_adding-the-vs-code-extension-using-the-che-plugins-panel.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// adding-che-plug-in-registry-vs-code-extension-to-a-workspace
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_adding-the-vs-code-extension-using-the-che-plugins-panel.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
+// it after the module has been published to ensure existing links are not
+// broken.
+//
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="adding-the-vs-code-extension-using-the-che-plugins-panel_{context}"]
+= Adding the VS Code extension using the Che Plugins panel
+
+To add the VS Code extension using the *Che Plugin* panel, follow the procedure below.
```

Comment:
I'd not have this line in the introduction to the procedure. It is not providing much information. Why not do it the way you did it for the Adding the VS Code extension using the workspace configuration section?

Reply:
Solved